data_bags/users/pablogsal.json

```diff
@@ -3,6 +3,9 @@
   "comment": "Pablo Galindo",
   "sudo": ["buildmaster"],
   "ssh_keys": [
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDIQDm0xnBIq4vAZd5VajdTUH14l7piki1IvPBo7tqKIqpEuEf/GgKDXGAsyBmVTiOHo8k0pTNtj/wmg4S3u00d5z+rPcrLNrHzyNEVjLwdpj4J+RTKubTO4McFEoSV9b8ApPkTujr6WhJT5G8woM+ZHLGpGQ1T1EbyNaLBKbC2pP2bVZ9eTQJhOedKXdYeq4trYnFZi6iNwVnsZY4BBboLy5T4JhmQJBMpPogtELnQ2rOA4EpkvY6Swcae5WpsxnkiF6qHbCjdQ2UlTVOwIQ9dAhysZ3179iZ8kJRCfV32LAQ45BfftBqk3Uhkumg67t81p/J3OsbjAfoQg/iU7N+5QSNrVCemk3N9isfDi1IuAUiPp09rSdETRmFt/yuGQuCVIQy1RXdmHrVuEYi5KzaiTL3dz6n8oKBxlS60Tx8DTj1Vy6MvvkTBJlZCMDsZ79ZpRvqmrVYP0j3OpPvH3zpMZx8RVG+lfggcx0tQASe2OxlbAUgBn1en3CT/URqnRilL98Dbn/5U7NS7I7hd+44HSQOwEEf3ULXSGlK4BUYoka2NVO8qa2b4+vHcQ6O8KoyUv3GHOf7Q7et3V0yBhMUG/9SClNpWjxlhD0UZ8NF36J6QejHd4AUjez/uFeMrD7ew51rUplioJL3h8WMiXVYmD/+4YQ== openpgp:0x28913191"
+    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC34fbzw9gfpohw0zAAyPGkWe4PbZ88p2WJJN/trAoorrHrbEEPhyDyfwEhCQezp2p3vXck/y00YayvamsYFTIppNgjlwf8coezJ8O5yYBVNGkUm7V9L270M9rLks60OvZjys2p9vCRqApagdG8Nb8H91Y48R1dnYA+ewpqhT7+5AAv9muenRPnrc12rbWQC/tiVk6Zn0NHj0UWnbQZtFWoUIZ1PMdPo7wFaxGsKqVVvN6M2Bclv9FiJzNie4x4ENejqLGOJY+S/w8aVcWgK4URaEX1j13wt+4hPDKLvyPeYrLz0CIadcE6OIxtCjNNgaCFW6f0ETQUOlraTs7ybNYKkIiSU36myiV0NzQxGPXK6VUvNxyfmb/fTzbMjZBYxkKVPOFN2YbCOMoI3WOOZqWbnqiNUVXEUHbvB4L6KP+/EP0oq+oDBqKd5lXVaNoLiJBXCa14dF5N3nIeGlaq5isf5j1xsDgEFNAwLpeEVt3yoNUMUBO05cN/veRHv/quYxwES9Lg2wiaUTvEpfDQxLi99p/9jrepEHgkh61OyiUWAJE8ZCr6/MzaMO4RYRxTxEpx7057JCOp6IdFWOjcDIZi74ZtMCpEcWT81DXN7IC72ytTcpiGfWigmWps4Wa5hr525RM/zki8BlLmuOuR94TNb7wUImq7Y+/umaliJEpgTw== cardno:000605651315",
```

Comment:
Looks like this one and the one on line 8 are duplicate; it should be safe to remove either one.